src/Parser/Druid/Balance/Modules/Items/LadyAndTheChild.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Wrapper from 'common/Wrapper';
+
+class LadyAndTheChild extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  moonfireCasts = 0;
+  moonfireHits = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasShoulder(ITEMS.LADY_AND_THE_CHILD.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (event.ability.guid !== SPELLS.MOONFIRE_BEAR.id || event.tick === true) {
+      return;
+    }
+      this.moonfireHits++;
+  }
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.MOONFIRE.id) {
+      return;
+    }
+      this.moonfireCasts++;
+  }
+
+  get percentTwoHits() {
+    return (this.moonfireHits - this.moonfireCasts) / this.moonfireCasts;
+  }
+
+  item() {
+    return {
+      item: ITEMS.LADY_AND_THE_CHILD,
+      result: (
+        <dfn data-tip={`You hit ${this.moonfireHits} times with ${this.moonfireCasts} casts.`}>
+          <Wrapper>{formatPercentage(this.percentTwoHits)}% of your <SpellLink id={SPELLS.MOONFIRE_BEAR.id}/> casts hit two targets.</Wrapper>
```

Comment:
While you don't have to implement this now, it would be nice to also see the damage gained due to LatC's moonfire damage buff.

Reply:
In a lot of situations the biggest DMG benefit is the saved GCD from not having to cast moonfire on the other target it hits so it isn't really possible to estimate the DPS gain. Showing only the increase from moonfire would be misleading in my opinion, that's why I left it out.

Reply:
It's pretty much impossible to estimate the gain because a big part of it is the saved GCDs in an AOE situation. On top of that Moonfire becomes the AOE filler instead of Solar Wrath/Lunar Strike when you have it equipped, so it would be inflated by that.

Reply:
I was suggesting more as part of the tooltip include an "also gained X% damage from buffed moonfire" ... but I can see actual rotation differences complicating accuracy.